feature-libs/quote/components/config/quote-ui.config.ts

```diff
@@ -14,6 +14,7 @@ import {
 
 export interface QuoteUIConfigFragment {
   maxCharsForComments?: number;
+  maxCharsForCardTile?: number;
```

Comment:
```suggestion
  truncateCardTileContentAfterNumChars?: number;
```
maxCharsForComments is really a limit, here we just configure the truncation, so we should name it differently.

Reply:
Okay! I see what you mean. I will adjust right now